### PR TITLE
chore(manual-release.sh): add

### PR DESCRIPTION
**Pull request by devopsmariocom**
manual release script

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/wc-catalogue/blaze-elements/blob/master/docs/CONTRIBUTING.md#commit
- [x] There are no linting errors and code is properly formatted (your run before push `yarn ts:format && yarn ts:lint:fix`)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
```

Add manual release script for cases like in #222 

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by Hotell** [comment]
just a heads up, these kind of scripts should be written cross-platform ( via node ). If I'm on windows I'll be having really hard time releasing manually. we can incorporate this in next PR ofc

**Comment by devopsmariocom** [comment]
@Hotell sure thing we should be multi platform first but this is just copy pasta from https://github.com/atlassian/lerna-semantic-release we don't want to spend development time on tools which we have to fix issues but rather fix root cause of these issues so these tools are not necessary ;)

**Comment by Hotell** [comment]
yup definitely, as I've said next PR or os if viable, cheers

File: toolbelt/manual-release.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+git pull
+
+pushd $RELATIVE_PKG
+npm version $VERSION_TYPE --no-git-tag-version
+VERSION=$(node -e "console.log(require('./package.json').version)")
+popd
+
+FULL_PACKAGE_NAME="$PKG@$VERSION"
+
+git commit -am"chore(release): manual bump for release of $FULL_PACKAGE_NAME\n\naffects: $FULL_PACKAGE_NAME"
```
Comment (hopkins-tk) [inline toolbelt/manual-release.sh]: Please remove affects.

**Review by hopkins-tk — APPROVED** [review]

**Pull request merged**